### ISSUE TITLE
Can you release your obfuscation map.

### ISSUE THREAD

**Issue by gameboys245** [THEREALWWEFAN231/Inertia#719]
**Minecraft Version(s)**
all

**What is the suggestion**
Can you release your obfuscation map.

**Why would this feature be useful**
You have said in the past that you will not release the source because you are scared that people will skid your mod and you wish at some point to sell your mod and its source. Why don't you release your obfuscation map. This is what Minecraft does. It allows you to still keep ownership of your code and allows decadently technical people to view how your mod works. With how your code is obfuscated now anybody with a good PC and the use of the tool [Enigma](https://github.com/FabricMC/Enigma) can decode and skid your mod and there is not much you can do about it. But although anybody can skid your mod people who want to understand it can't. [Enigma](https://github.com/FabricMC/Enigma) is a tool by FabricMC that allows anybody to decode jars obfuscated or not. It shows how the Byte-code of a jar would look in java, this allows easy copy and paste with the same affect as the original source but makes it hard to read.

On the flip side if you were to release your obfuscation map. People could read your code. And the prosses to skid your mod would not change and you could as you can do now you load up Inertia in [Enigma](https://github.com/FabricMC/Enigma) and set the Decompiler to CFR and copy and paste. But if you release your obfuscation map people who actually know what they are doing can load up a obfuscation map and read what it means. A pubic obfuscation map WOULD NOT increase the skid-ing but it would increase understanding. 

In conclusion if you released your obfuscation map you would keep the licence to your code and the right to sell it. It would not make it easier to skid your code as the process would stay the some obfuscation map or not. And the type of people who skid are lacking in the knowledge required it decode a jar with a obfuscation map.

**Comment by THEREALWWEFAN231** [comment]
Bruh, ill read the rest soon, but this does not get around the facts i states when "You have said in the past that you will not release the source because you are scared that people will skid your mod and you wish at some point to sell your mod and its source".

**Comment by THEREALWWEFAN231** [comment]
"A pubic obfuscation map WOULD NOT increase the skid-ing but it would increase understanding". That is false, as soon as people can get their hands on the code, even if they have to change it a little(to make it work), they will "steal" it(and or parts of it). Also currently engima isn't *really* capable of loading a obfuscation map for inertia, as there are a couple more *advance/basic* obfuscation techniques used. Which stop the majority of "skidders". Lastly lets say i did this, "In conclusion if you released your obfuscation map you would keep the licence to your code and the right to sell it". Who would buy something when its already basically fully available. Why would i buy diamonds from someone, when theres a literally public diamond mine a block down the street?

**Comment by THEREALWWEFAN231** [comment]
Also I dont know if this is related to #707 I assume it is, The only reason I could see someone wanting the source so badly is to copy a couple modules and or see how they work so they can "make"/edit it them selves. I understand some people do generally want to know how some stuff works but most the people(And i think you, I dont know who you are but I'm just assuming) want to know how it works so they can add it to their mod or what ever. You just have to really think how stuff works, how does note bot work, how does the color picker in the color manager work, how does strong hold finder work, how does teleport finder work, everything the client(your game instance) knows comes from a packet, so for some information you can just read a packet, on other stuff a google search will give you the math for it. For other stuff you may have to think a lot more.

**Comment by GaiaAlexf** [comment]
yea things like stronghold finder are just math... and yet people want the code bc they think it's magic when it's only triangulation
